src/Avalonia.Controls/MenuItem.cs

```diff
@@ -341,7 +342,7 @@ public string? GroupName
         /// <inheritdoc/>
         IMenuElement? IMenuItem.Parent => Parent as IMenuElement;
 
-        protected override bool IsEnabledCore => base.IsEnabledCore && _commandCanExecute;
+        protected override bool IsEnabledCore => base.IsEnabled && (HasSubMenu || _commandCanExecute);
```

Comment:
Is the change from `base.IsEnabledCore` to `base.IsEnabled` wanted? The logic stays the same (currently `base.IsEnabledCore` returns `IsEnabled`), but it seems a little odd to not refer to the base property.